lib/puppetserver/utils/http_client.rb

```diff
@@ -62,6 +62,15 @@ def put(body, url_override = nil)
           Result.new(result.code, result.body)
         end
 
+        def get(url_overide = nil)
```

Comment:
I think we have 2 "gets" now.  (See line 48 above...sorry i did that)  So you should be able to add the Result.new to that.

Reply:
Yep, I missed that.